backend/dataall/modules/omics/cdk/pivot_role_omics_policy.py

```diff
@@ -0,0 +1,45 @@
+from aws_cdk import aws_iam as iam
+from dataall.core.environment.cdk.pivot_role_stack import PivotRoleStatementSet
+
+
+class OmicsPolicy(PivotRoleStatementSet):
+    """
+    Creates an Omics policy for Pivot role accessing and interacting with Omics Projects
+    """
+
+    def get_statements(self):
+        return [
+            iam.PolicyStatement(sid='OmicsList', actions=['omics:List*'], resources=['*']),
+            iam.PolicyStatement(
+                sid='OmicsWorkflowActions',
+                actions=['omics:GetWorkflow', 'omics:StartRun', 'omics:TagResource'],
+                resources=[
+                    f'arn:aws:omics:{self.region}:{self.account}:workflow/*',
+                    f'arn:aws:omics:{self.region}::workflow/*',
+                    f'arn:aws:omics:{self.region}:{self.account}:run/*',
```

Comment:
can we restrict to resource_prefix for omics run ARNs?

Reply:
Also does pivotRole need `omics:StartRun` ? Believe runs are started by env group IAM Roles

Reply:
Hi Noah! Yes, I agree that we should actually try removing the 'OmicsWorkflowActions' and 'OmicsWorkflowActions' policy statements from the pivot role. These are duplicated in the env role. As you know, IAM permissions are tricky, so we were more permissive in the development phase. We will test removing these and report back in a comment here.